docs/abstract.md

```diff
@@ -0,0 +1,3 @@
+## pServer Abstract
+
+The purpose of this project is to store secret strings securely on a server. The server keeps an internal registry of users and their connection to certain private data, as well as a collection of their public keys. Users can have permission to read and write to files on the server, or see them listed. 
```

Comment:
I would change `permission to read and write to files on the server` to `permission to read and write to secured strings on the server`.

Reply:
Updated.